common/src/main/java/com/microsoft/identity/common/internal/net/AbstractHttpClient.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.identity.common.internal.net;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+
+public abstract class AbstractHttpClient implements HttpClient {
+
+    @Override
+    public HttpResponse method(@NonNull String httpMethod, @NonNull URL requestUrl, @NonNull Map<String, String> requestHeaders, @Nullable byte[] requestContent) throws IOException {
+        return method(HttpClient.HttpMethod.validateAndNormalizeMethod(httpMethod), requestUrl, requestHeaders, requestContent);
+    }
```

Comment:
Is this method being used anywhere (after your changes)?

Reply:
Probably.  I just moved it from the non-abstract class to the abstract class.  HttpRequest.sendWithMethod delegates here.